configure.ac

```diff
@@ -371,7 +371,7 @@ AC_SUBST(PLUGIN_CFLAGS)
 AC_PROG_CC
 AM_PROG_CC_C_O
 
-AC_LIB_PREFIX
+#AC_LIB_PREFIX
```

Comment:
Why commented this out? What is the impact of not having it?
Also note that from https://wiki.strongswan.org/issues/652 installing some extra packages should solve issues with this macro

Reply:
`error: possibly undefined macro: AC_LIB_PREFIX`